### DISCUSSION TITLE
Having Issues Launching Boilerplate

### DISCUSSION THREAD

**Discussion by easternlai** [NoQuarterTeam/boilerplate-old#57] [General]
hello!

Huge fan of the repos you all have been sharing. I was trying to launch this new boilerplate.  I ran "npm i" from the root directory then added the DATABASE_URL to a .env in the root as well.  when i try to run "pnpm db:push in packages/database I get a command failed error.  

Is there a plan at some point to make the README file as descriptive as the README in the boilerplate-graphql? 

Thanks so much for all you do 👍

**Comment by JClackett** [comment]
Hey! Thanks :) 

I think it may be because this repo currently uses pnpm, and so has a pnpm lock file. So when running the pnpm command, unless you have that installed and you have run pnpm install in the project, it probably won't work. 

There are a few things related to pnpm in this repo that allow it all to work (workspaces, hoisting etc) so im not 100% sure if it will just work with npm.

**Comment by easternlai** [comment]
Appreciate the answer, just had another quick question. I notice there is a Next folder in Apps, curious what the Next.js app will be used for it this boilerplate?

**Reply by JClackett** [reply]
I want to be able to pick and choose between next and remix as they each have their own benefits - however next is still too far behind in terms of making real world apps still for me. They don't have a recommended approach to auth and a few other things like infinite pagination, but once that all gets sorted it will be a bit closer. The RSC story is still too early to take the risk especially with paying clients!
